pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/JsonAsyncHttpPinotClientTransport.java

```diff
@@ -67,9 +67,16 @@ public BrokerResponse executeQuery(String brokerAddress, String query)
   public Future<BrokerResponse> executePinotQueryAsync(String brokerAddress, final Request request) {
     try {
       ObjectNode json = JsonNodeFactory.instance.objectNode();
-      json.put(request.getQueryFormat(), request.getQuery());
-
-      final String url = "http://" + brokerAddress + "/query";
+      String queryFormat = request.getQueryFormat();
+      json.put(queryFormat, request.getQuery());
+
+      final String url;
+      if (queryFormat.equalsIgnoreCase("sql")) {
```

Comment:
I feel this is not very intuitive as to why the url path has /query for PQL query format and /sql for SQL query format. May be both modes can have query prefix? I don't think we can change the current url format as that would be backward compatible but for the SQL format, I think we should consider having /query/sql

Reply:
I had considered "/query/sql". But felt it was okay to go with just "sql", as we would eventually be getting rid of the "/query" endpoint.
I don't have a strong preference either way. I can change it if majority think this one is better. I got a signoff on the issue #4962 for "/sql"

Reply:
changed thi to /query/sql